cypress/integration/org_events_page.spec.ts

```diff
@@ -5,10 +5,14 @@ describe('/o/[orgId]/events', () => {
         cy.contains('My Organization');
     });
 
-    xit('contains events', () => {
+    xit('contains events which are linked to event pages', () => {
         // TODO: Figure out why this fails on GitHub
         cy.visit('/o/1/events');
         cy.get('[data-test="event"]').should('be.visible');
+        cy.get('[data-test="more-info-button-22"]')
+            .should('be.visible')
+            .click();
+        cy.url().should('match', /\/o\/1\/events\/[0-9]+$/);
```

Comment:
> Looking at the wireframes, there are no buttons in the event list. Maybe we need to revisit the wireframes, and I think there is merit in having a sign-up button right there in the list, but do we need one for "More info"?
> 
> I think it should be possible to click the title of the event to go to the event page.
> 
> I also think that if we do keep a spearate "More info" link around, it should be a "secondary action" and not use the same prominent button widget that we use for sign up. What do the guidelines look like for a scenario like this in Adobe Spectrum?

This is a relevant discussion! I agree a sign-up button id good to keep in place. Regarding the more info button I tried making a link for the top level event item element, but this entailed modification of css for `<a>` tags which felt a bit outside of the scope of the issue (?). Since I was unsure of where else to put the link, based on the wireframes, I decided to add a button. In regards to visual hierarchy between the buttons I believe this is maybe a bit unnecessary before we get UI design from Gemma? The main decission I see here is to choose between link on title or button (or event item element?).